drivers/comparator/Kconfig.stm32_comp

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2025 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config COMPARATOR_STM32_COMP
+	bool "ST STM32 comparator driver"
+	default y
+	depends on DT_HAS_ST_STM32_COMP_ENABLED
+	select PINCTRL
+	select USE_STM32_LL_COMP
+	select USE_STM32_LL_EXTI
+	select USE_STM32_LL_PWR if SOC_SERIES_STM32H7X
+	select USE_STM32_LL_RCC if SOC_SERIES_STM32H7X
```

Comment:
Note that use of these symbols is required only if you're making use of LL source files.
They are not required if you only use headers.

Reply:
@erwango  Thank you. I've already realized it from cmake target binding in corresponding cmake file. But anyway it was a very good one note - I wanted to review this section. Truly both LL_PWR and LL_RCC were not used s.t. .c files wer built and linked. Now I dropped both settings. All the tests still passing (at least locally).